src/app/dashboard-widgets/applications-widget/applications-widget.component.spec.ts

```diff
@@ -201,7 +221,28 @@ describe('ApplicationsWidgetComponent', () => {
 
           return mock;
         }
-      }
+      },
+      {
+        provide: Router, useFactory: (): jasmine.SpyObj<Router> => {
+          let mockRouterEvent: any = {
+            'id': 1,
+            'url': 'mock-url'
+          };
+
+          let mockRouter = jasmine.createSpyObj('Router', ['createUrlTree', 'navigate', 'serializeUrl']);
+          mockRouter.events = Observable.of(mockRouterEvent);
+
+          return mockRouter;
+        }
+      },
+      {
+        provide: UserService, useFactory: () => {
+          let userService = createMock(UserService);
+          userService.getUser.and.returnValue(fakeUser);
+          userService.loggedInUser = fakeUser.publish() as ConnectableObservable<User> & jasmine.Spy;
+          return userService;
+        }
+      },
```

Comment:
Trailing comma; I think the tslint rule is not to have these but I could be wrong.

Reply:
```
$ npm run lint

> fabric8-ui@0.0.0-development lint /fabric8-ui
> find src -name '*.ts' | xargs tslint -p . -c tslint.json --force


ERROR: fabric8-ui/src/app/dashboard-widgets/applications-widget/applications-widget.component.spec.ts[10, 9]: missing whitespace
ERROR: fabric8-ui/src/app/dashboard-widgets/applications-widget/applications-widget.component.spec.ts[10, 39]: missing whitespace
ERROR: fabric8-ui/src/app/dashboard-widgets/applications-widget/applications-widget.component.spec.ts[245, 8]: Unnecessary trailing comma
ERROR: fabric8-ui/src/app/dashboard-widgets/applications-widget/applications-widget.component.spec.ts[250, 53]: Spaces before function parens are disallowed
ERROR: fabric8-ui/src/app/dashboard-widgets/applications-widget/applications-widget.component.spec.ts[254, 47]: Spaces before function parens are disallowed
ERROR: fabric8-ui/src/app/dashboard-widgets/applications-widget/applications-widget.component.spec.ts[260, 53]: Spaces before function parens are disallowed
ERROR: fabric8-ui/src/app/dashboard-widgets/applications-widget/applications-widget.component.spec.ts[266, 51]: Spaces before function parens are disallowed
ERROR: fabric8-ui/src/app/dashboard-widgets/applications-widget/applications-widget.component.spec.ts[272, 52]: Spaces before function parens are disallowed
ERROR: fabric8-ui/src/app/dashboard-widgets/applications-widget/applications-widget.component.spec.ts[276, 50]: Spaces before function parens are disallowed
ERROR: fabric8-ui/src/app/dashboard-widgets/applications-widget/applications-widget.component.spec.ts[283, 68]: Spaces before function parens are disallowed
ERROR: fabric8-ui/src/app/dashboard-widgets/applications-widget/applications-widget.component.spec.ts[291, 77]: Spaces before function parens are disallowed
ERROR: fabric8-ui/src/app/feature-flag/resolver/feature-flag.resolver.spec.ts[3, 9]: missing whitespace
ERROR: fabric8-ui/src/app/feature-flag/resolver/feature-flag.resolver.spec.ts[3, 43]: missing whitespace
ERROR: fabric8-ui/src/app/shared/owner-guard.service.spec.ts[5, 1]: Import sources within a group must be alphabetized.
```

While you're in this area, can you fix the tslint issues? The issues in other areas (feature-flags, etc.) can be fixed in another PR (not necessarily written by yourself)

Reply:
Will do

Reply:
Fixed them all